grin/test/Transformations/Optimising/CopyPropagationSpec.hs

```diff
@@ -76,7 +76,7 @@ spec = do
         |]
       copyPropagation (ctx before) `sameAs` (ctx after)
 
-    it "right unit law" $ do
+    xit "right unit law" $ do
```

Comment:
Why?

Reply:
Reenabled with a minor modifications.

Reply:
Slightly modified the expected result and reenabled the test case.